docs/doc_assets/js/start.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var $ = require('jquery');
+window.$ = window.jQuery = require('../../../node_modules/jquery/dist/jquery');
```

Comment:
Should we just reference `require('jquery');` here rather than the whole path? Otherwise it looks good. :100: